src/arm64/bitmanipulation.h

```diff
@@ -60,14 +60,13 @@ really_inline bool add_overflow(uint64_t value1, uint64_t value2, uint64_t *resu
 }
 
 #ifdef _MSC_VER
-#pragma intrinsic(_umul128) // todo: this might fail under visual studio for ARM
```

Comment:
Microsoft's documentation always include "#pragma intrinsic(_umul128)". It is being removed from the code in this PR. Why? Either it is unnecessary, or else...?

Reply:
Note: I realize that the intrinsic may only be available under x64. Still... do we need the pragma?

Reply:
I think your own code should never add this pragma (in this and all other places). You should only include proper headers to use it.

Reply:
Do you have a reference supporting this statement? All I could find online is this...

"The intrinsic pragma tells the compiler that a function has known behavior. The compiler may call the function and not replace the function call with inline instructions, if it will result in better performance."

I'll ask on twitter.

Reply:
Note that Microsoft's documentation gives examples where both the header is included and the pragma called https://docs.microsoft.com/en-us/cpp/intrinsics/umul128?view=vs-2019

I expect I will know more soon.

Reply:
Yes, that's weird that they put it that way in the examples. AFAIK, intrin.h already does define it as intrinsic, and compiler obviously knows that it's intrinsic

Reply:
In any case, pragma thing is irrelevant for the PR: function is changed not to use `_umul128` here, so, the pragma was removed also.